spec/requests/api/v1/student_tasks_controller_spec.rb

```
@@ -100,59 +103,165 @@
+        type: :object,
+        properties: {
+          response_map_id: { type: :integer }
+        },
+        required: ['response_map_id']
+      }
+
+      response '200', 'task started or blocked by queue ordering' do
+        let(:body) { { response_map_id: review_map.id } }
+        run_test! do |response|
+          expect([200, 403]).to include(response.status)
+        end 
```

RuboCop (1.86.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)